### ISSUE TITLE
Inconsistency between Firefox and Chromium

### ISSUE THREAD

**Issue by Kyuunex** [rebane2001/mapartcraft#292]
I recently switched to Firefox from Chromium and later found myself in a need to rebuild a previously built schematic to swap out Dirt with Jungle Planks. This means using the exact same settings again then changing what I need. I always take screenshots of my settings for this purpose.

This time I noticed that I was unable to recreate my schematic this way. After further testing, I realized it was indeed due to inconsistency between Firefox and Chromium, with Chromium in my opinion leading to better results, but that could just be me having spent very long time tweaking this image in Chromium.

Palette is this https://rebane2001.com/mapartcraft/?preset=0Q1R4R8R9QbRcRdReRfRgRhRiRjRkRlRmRnRoRpRqRrRsRxRyQzQ10Q11Q12Q13Q14Q15Q16Q17Q18Q19Q1aQ1bQ1cQ1dQ1eQ1fQ1gS1iQ1jS1mR

Observe the settings:
![2024-07-13T18:16:25+00:00](https://github.com/user-attachments/assets/dfba359c-350d-440a-a6d4-6aea9600dc92)

Observe what Chromium generated:
![ayanami_v2_chromium](https://github.com/user-attachments/assets/21458489-13eb-4e83-a3fe-cf9cefb241e0)

Observe what Firefox generated:
![ayanami_v2_firefox](https://github.com/user-attachments/assets/fde30fbf-fc68-43ac-8e3c-91ac5a0334e9)

I believe these generations should be consistent, but they are not.

This is the source image:
![89932952_p0](https://github.com/user-attachments/assets/b9fa3247-033d-4e69-a2be-68eb9a39ec54)

**Comment by rebane2001** [comment]
Does the difference still exist once you resize the image to the same resolution as your map?

**Comment by Kyuunex** [comment]
Yes. I just resized the source image to 512x768 and tried testing again. The difference is still there.

**Comment by koboshchan** [comment]
can you point out the difference by the way @Kyuunex
